src/test/resources/data/FeedbackSessionQuestionTypeTest.json

```diff
@@ -868,6 +868,28 @@
         "INSTRUCTORS"
       ]
     },
+    "qn4InSession4InCourse1": {
```

Comment:
I think there are existing questions for `constsum`?

Reply:
Yes there are 3 existing questions. One of them is `Student -> self` (self evaluation). I can't use self evaluation for this test as we need more than one entities to give feedback to, so self evaluation questions will not work.

one question is `Instructors -> TEAM` and the third one is `Instructors -> STUDENTS`. If I edit any of them I will have to refactor every test that uses these questions.

Reply:
@xpdavid 
There are some existing questions in other json files that I guess we can use. Should I use them?

(One thing that I will like to point out is, the existing questions I mentioned above belong to `distribute points to recipients` category, the question I have added in this test is `distribute points to options` category, As we already have the `recipient` case covered in `InstructorFeedbackSubmissionEditSaveActionTest`, I was thinking if it would be a good idea to have the `distribute points to options` case covered in the student case?)

Reply:
That would be good. I think your fix is fixing all question type so you may want to test different question types.

Reply:
@xpdavid 

>I think your fix is fixing all question type so you may want to test different question types.

Just to be clear, I should add tests for more question types in this test only right? Or, should i create different `@Test` for different question types? 

>That would be good.

By this you mean I should keep the question I added right? Or, are you suggesting to use an existing question of some other question type instead of `distribute points to options` question?

Reply:
Maybe just one test using the existing question should be enough for student side.